graph/api/src/main/java/org/jboss/windup/graph/service/ProjectService.java

```diff
@@ -67,4 +78,62 @@ public ProjectModel getOrCreateSharedLibsProject()
 
         return sharedLibsProject;
     }
+
+    public Map<ProjectModel, ProjectModel> getProjectToRootProjectMap()
```

Comment:
Ah, right. I missed this one, but I think you are right. Calling getRootProject should be enough.